examples/trials/mnist-tfv2/config_pai.yml

```diff
@@ -0,0 +1,32 @@
+authorName: NNI Example
+experimentName: MNIST TF v2.x
+trialConcurrency: 1
+maxExecDuration: 1h
+maxTrialNum: 10
+#choice: local, remote, pai
+trainingServicePlatform: pai
+searchSpacePath: search_space.json
+#choice: true, false
+useAnnotation: false
+tuner:
+  #choice: TPE, Random, Anneal, Evolution, BatchTuner, MetisTuner, GPTuner
+  #SMAC (SMAC should be installed through nnictl)
+  builtinTunerName: TPE
+  classArgs:
+    #choice: maximize, minimize
+    optimize_mode: maximize
+trial:
+  command: python3 mnist.py
+  codeDir: .
+  gpuNum: 0
+  cpuNum: 1
+  memoryMB: 8196
+  #The docker image to run nni job on pai
+  image: msranni/nni:latest
```

Comment:
Tensorflow 1.x is installed in msranni/nni right?

Reply:
I think so. Because nobody has upgraded it.

Reply:
So is it appropriate for us to put an example here that doesn't even work?

Reply:
I removed docker-based config files. Now this example does work, but only supports limited training services.

Reply:
Because distributed platforms are not supported, I moved this example down in docs...